App/Sagas/NodeLifecycle.ts

```diff
@@ -37,6 +38,9 @@ export function * manageNode () {
       // or by launching into the background because of a trigger.
       if (action.payload.newState === 'background') {
         yield fork(backgroundTaskRace)
+      } else {
+        // Check for new photos on every Foreground event
+        yield put(StorageActions.refreshLocalImagesRequest())
```

Comment:
This will work most of the time, but it's not guaranteed to work. Refreshing the local images eventually relies on the node being created/started/online, but you're refreshing the local images here without any guarantee that the node will be ready.